lib/qwerty/quran_simple.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'text'
+require_relative 'config'
+module Qwerty
+  class Text
+    class QuranSimple < Ruote::Participant
+      def on_workitem
+          workitem.fields['text'][:quran_simple] = {
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.